src/App.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import "./App.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+import Header from "./view/Header";
+import Main from "./view/Main";
+import Footer from "./view/Footer";
+
+class App extends Component {
+	render() {
+		return (
+			<div className="App">
+				<Header />
+				<Main />
+				<Footer />
+			</div>
+		);
+	}
+}
```

Comment:
Sorry for the nitpicking about this but I think this tabulation is too much. In my opinion 2 or 4 spaces are enough.

check this example:
https://github.com/gothinkster/react-redux-realworld-example-app/blob/master/src/components/Editor.js#L95